### ISSUE TITLE
cURL Error: Operation timed out after 15000 milliseconds with 0 bytes received

### ISSUE THREAD

**Issue by kantsverma** [Codexshaper/laravel-woocommerce#69]
Getting these issues on the server after deploying the code. It's working fine in localhost.

Even I am adding the dummy products ie

`			$data = [
				'name' => 'TBI Test Product',
				'type' => 'simple',
				'regular_price' => '10.00',
				'description' => 'Simple product full description.',
				'short_description' => 'Simple product short description.',
				'categories' => [
					[
						'id' => 1
					]
				],
				'images' => [
					[
						'src' => 'http://demo.woothemes.com/woocommerce/wp-content/uploads/sites/56/2013/06/T_2_front.jpg'
					],
					[
						'src' => 'http://demo.woothemes.com/woocommerce/wp-content/uploads/sites/56/2013/06/T_2_back.jpg'
					]
				]
			];
			
			$product = Product::create($data);`

**Comment by maab16** [comment]
Hello @kantsverma,

I think it's WordPress related issue. Please check your store server and you can increase the PHP memory limit and execution time

You can use `ini_set`

```
ini_set('memory_limit', '-1'); // Unlimited
ini_set('max_execution_time', '0'); //Unlimited
```

After execution script then reset default memory_limit and execution.

**Comment by kantsverma** [comment]
@maab16 yes i resolved it thank you for the response.

**Comment by maab16** [comment]
You can share the solution how you solve this problem so that other guys
don't waste their time.

Here we are all help each other. Thanks. 🙂

**Comment by kantsverma** [comment]
@maab16  Yes sure I was sending the wrong parameters of the Product image. All the parameters should be perfect even the product image URL should be propper image URL that Woocomerce can copy to the WP upload folder as a product image.

**Comment by YassineChe** [comment]
@kantsverma , Didn't works for me, when i decrease number of image URLs works fine,
even i changed `WOOCOMMERCE_WP_TIMEOUT` in `woocommerce.php`
any suggestions ?
